kitchen-tests/kitchen.dokken.yml

```diff
@@ -14,17 +16,27 @@ provisioner:
     diff_disabled: true
     always_dump_stacktrace: true
   chef_license: "accept-no-persist"
+  chef_license_key: free-58e5ca65-d1cc-4838-bb79-42aa40ed3623-3835
```

Comment:
Should this be stored in a vault or is it ok if this is loose to humanity?

Reply:
This key has been approved to be in the wild due to secrets in GHA not being transferable nor work on community forked PR's back to our main.